server/apps/research/urls.py

```
@@ -5,6 +5,8 @@
 from rest_framework.routers import DefaultRouter
 from .views import ArticleViewSet
 from .redirects_urls import urlpatterns as redirects_urlpatterns
+from django.urls import path
```

Ruff (0.8.2)

Redefinition of unused `path` from line 1

Remove definition: `path`

(F811)